src/encoding.ts

```diff
@@ -322,11 +322,9 @@ export function extractTransformsFromEncoding(oldEncoding: Encoding<Field>, conf
           // Add formatting to appropriate property based on the type of channel we're processing
           const format = getDateTimeComponents(timeUnit, config.axis.shortTimeLabels).join(' ');
           if (isNonPositionScaleChannel(channel)) {
```

Comment:
`isLegendScaleChannel` does not exist but we should use something equivalent here or move the check for text and tooltip up.